frontend/src/time_utils.js

```diff
@@ -7,3 +7,20 @@ export function convertTimestampToDate(timestampDate) {
     timestampDate.date.year, timestampDate.date.month - 1, timestampDate.date.day, 0, 0, 0, 0)
     .toDateString();
 }
+
+const DATE_DELIMITER = '/';
+
+/** Format JavaScript date in the form MM/DD/YYYY if mFirst is true, otherwise YYYY/MM/DD. */
+export function formatDate(date, mFirst) {
```

Comment:
monthFirst (this is from the other PR, probably ude to some out-of-sync thing

Reply:
yeah it's out-of-sync i changed it to monthFirst in the other one